### PR TITLE
Project card separator line

### PR DESCRIPTION
**Pull request by rukawa444**
Fixed project card separator line.
line

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-2)

**Comment by rukawa444** [comment]
> Go go go 😄

Thank you!

File: cypress.json [hunk 1]
```diff
@@ -1 +1 @@
-{}
+{ "projectId": "1fuai2" }
```
Comment (jkettmann) [inline cypress.json:1]: Just out of curiosity: was there a specific reason this was added?

**Comment by jkettmann** [comment]
Great way of writing the PR descriptions btw 😄

File: cypress.json (hunk 1 above)
Comment (rukawa444) [reply cypress.json:1]: Hey Johannes!

Every time I push the changes, the CI pipeline pointed out that Cypress has failed and couldn't figure out what was wrong as I only edited the bug. then tried to fix the cypress thing and that's why I added it and CI run successfully.

**Comment by rukawa444** [comment]
I learned it through the minesweeper tho lol! 😎

regarding that, mentor Tara, doesn't do any review but the CI pipeline always says it passed and ready to merge. so I always continue to squash and merge. never heard from her lol! And now, I wanted to play it again to practice more but couldn't because she can't give me the new game. haha. but it was fun!

File: cypress.json (hunk 1 above)
Comment (rukawa444) [reply cypress.json:1]: But I've edited it out now, run again and it behaved well with my last commit.

Comment (jkettmann) [reply cypress.json:1]: One of the Cypress tests is a bit flaky unfortunately. Sometimes it helps to just re-run the CI jobs. But no idea if that was the problem

**Pull request merged**